source/_includes/links.md

```diff
@@ -285,7 +285,9 @@
 [registry-profiles]: {{ site.api_url | absolute_url }}/registry/profiles/
 [registry-timeModes]: {{ site.api_url | absolute_url }}/registry/timeModes/
 [registry-types]: {{ site.api_url | absolute_url }}/registry/types/
-[registry-viewingDirections]: {{ site.api_url | absolute_url }}/registry/viewingDirections/
+[registry-viewingDirections]: {{ site.api_url | absolute_url }}/registry/
+viewingDirections/
```

Comment:
I'm not sure how this line break got in here. It's not a new line in my local version

Reply:
fixed